homeassistant/components/philips_js/media_player.py

```diff
@@ -83,7 +84,7 @@ def __init__(
         self._sources: dict[str, str] = {}
         self._supports = SUPPORT_PHILIPS_JS
         self._system = coordinator.system
-        self._attr_name = coordinator.system["name"]
+        self._attr_name = None
```

Comment:
This is the default, this line can be removed.

Reply:
```suggestion
```